apps/jubmoji-quest/src/pages/quests/[id].tsx

```diff
@@ -243,10 +248,10 @@ export default function QuestDetailPage() {
                         {quest.proofType ===
                           $Enums.ProofType.TEAM_LEADERBOARD &&
```

Comment:
We should probably have another quest DB column that determines whether the collection is hidden on frontend; this isn't actually tied to team leaderboard but was just a hack for the hunt

Reply:
https://github.com/jubmoji/jubmoji.quest/issues/167